### ISSUE TITLE
Bugs with AbstractAdmin::getParentAssociationMapping

### ISSUE THREAD

**Issue by erik-zicht** [zicht/page-bundle#60]
Introducing the second argument for `addChild` as shown here https://github.com/zicht/page-bundle/commit/074aeb5e1d4801e23d2ccf415d8f50966f422472#diff-d0988d5d97da3c245980abbe4e500638R103 enables the parentAssociationMapping and all of its related code. 

This causes bugs: contentitems checking the property `contentItems` on itself.

```
An exception has been thrown during the rendering of a template ("Neither the property "contentItems" nor one of the methods "getContentItems()", "contentItems()", "isContentItems()", "hasContentItems()", "__get()" exist and have public access in class "App\Entity\ContentItem\Text".").
```

**Comment by erik-zicht** [comment]
Possibly related with how we setup adding and editing contentitems in the projects using this bundle. We have `ContentItemAdmin` for adding contentitems, the projects also introduce a `ContentItemDetailAdmin` for editing.

**Comment by 7ochem** [comment]
Resolved in #89
